bin/cron/zendesk_slack_report

```diff
@@ -267,4 +259,3 @@ end
 #
 #
 get_tickets_by_group
-get_tickets_by_user
```

Comment:
This functionality doesn't make as much sense in our new Zendesk setup but I'm leaving the function itself behind just in case.